account/src/main/java/com/caldevsupplychain/account/service/AccountService.java

```diff
@@ -0,0 +1,50 @@
+package com.caldevsupplychain.account.service;
+
+
+
+import com.caldevsupplychain.account.model.Role;
+import com.caldevsupplychain.account.model.User;
+import com.caldevsupplychain.common.bean.account.UserBean;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
+public interface AccountService {
+
+    boolean userExist(String emailAddress);
+
+    /****************************************************************
+     *                      User                                    *
+     ****************************************************************/
+    User createUser(UserBean userBean);
```

Comment:
return `UserBean` instead of `User` for all methods in this service.
`UserBean createUser(UserBean userBean);`